### PR TITLE
23/OKTA Integration in React

### PR DESCRIPTION
**Pull request by manjeetbeniwalinnostax**
[Issue#23](https://github.com/Innostax/tavajs/issues/23)

### CODE REVIEW

File: src/authTemplates/oktaTemplate/AppWithRouterAccess.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import Home from './Home'
+import SignIn from './SignIn'
+
+const AppWithRouterAccess = () => {
+	const history = useHistory()
+	const onAuthRequired = () => {
+		history.push('/login')
+	}
+	console.log('enviorment ', process.env)
+	const CLIENT_ID = process.env.REACT_APP_OKTA_CLIENT_ID
```
Comment (manjeetbeniwalinnostax) [inline src/authTemplates/oktaTemplate/AppWithRouterAccess.jsx:14]: Remove console

File: src/authTemplates/oktaTemplate/SignIn.jsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { Redirect } from 'react-router-dom'
+import SignInForm from './SignInForm'
+import { useOktaAuth } from '@okta/okta-react'
+
+const SignIn = () => {
+	const { authState } = useOktaAuth()
+
+	if (!authState) {
```
Comment (manjeetbeniwalinnostax) [inline src/authTemplates/oktaTemplate/SignIn.jsx:9]: Remove extra line break

File: src/authTemplates/oktaTemplate/SignInForm.jsx [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+	const { oktaAuth } = useOktaAuth()
+	const [sessionToken, setSessionToken] = useState()
+	const [username, setUsername] = useState('')
+	const [password, setPassword] = useState('')
+
+	const handleSubmit = (e) => {
+		e.preventDefault()
+
+		oktaAuth
+			.signInWithCredentials({ username, password })
```
Comment (manjeetbeniwalinnostax) [inline src/authTemplates/oktaTemplate/SignInForm.jsx:14]: Format file properly

File: src/authTemplates/oktaTemplate/SignInForm.jsx [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+			.then((res) => {
+				const sessionToken = res.sessionToken
+				setSessionToken(sessionToken)
+				// sessionToken is a one-use token, so make sure this is only called once
+				oktaAuth.signInWithRedirect({ sessionToken })
+			})
+			.catch((err) => console.log('Found an error', err))
+	}
```
Comment (manjeetbeniwalinnostax) [inline src/authTemplates/oktaTemplate/SignInForm.jsx:22]: use console.warn instead of console.log for errors

**Review by himanshuinnostax — APPROVED** [review]

**Pull request merged**